index.html

```diff
@@ -4,34 +4,52 @@
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>TanmayiPortfolio</title>
+    <title>Tanmayi'sPortfolio</title>
     <link rel="stylesheet" href="styles.css" type="text/css">
 </head>
 <body>
     <!--Outer div consists toolbar and headline-->
     <div class="wrapper">
         <div class="main-background">
             <!--section one toolbar-->
-            <header class="toolbar">
+			<div id="menudrop" class="menudrop"></div>
+			<div id="menu-outline" class="menu-outline">
+				<button id="closebtn">
+				  <div class='bar11'></div>
+				  <div class='bar22'></div>
+				</button>
+				<ul class='main-Menu'>
+					<li><a class='menuItem' href="#portfolio">Portfolio</a></li>
+					<li><a class='menuItem' href="#about">About</a></li>
+					<li><a class='menuItem' href="#contact">Contact</a></li>
+				</ul>
+			</div>
+            <header class="toolbar" id='header'>
                 <nav class="logo-hamburger">
-                <a href='#' class="logo"><span>T</span></a>
-                <a href='#' class="hamburger"><img src="./media/hamburger.png" alt="menu"></a>
+                    <a href='#' class="logo"><span>T</span></a>
+                    <button class="hamburger" id='hamburger'>
+                        <div class='bar1'></div>
+						<div class='bar2'></div>
+						<div class='bar3'></div>
+                    </button>
+					
                 </nav>
```

Comment:
very well done so far, but when the mobile menu is open and we click on a menu item the mobile header takes a position at the bottom and when we click on hamberger btn it doesn't show the menu again kindly fix this issue you can try adding event listners to menu items to close menu when clicked  also you can give the header a fixed position on top 

![image (28)](https://github.com/tan12082001/Project-portfolio/assets/59707859/d4b2623f-7e47-4b50-8176-a518610722a6)
![image (29)](https://github.com/tan12082001/Project-portfolio/assets/59707859/14e7512a-4f0e-4586-8f57-6b9546fd148c)